Assignment/HW5_SQL/HW5_Starter_Code/app/models.py

```diff
@@ -1,13 +1,51 @@
 import sqlite3 as sql
 
-def insert_data():
-    # SQL statement to insert into database goes here
+def insert_data(first_name, last_name, company, email, phone, street_address, city, state, country, zip_code):
+	# SQL statement to insert into database goes here
+	with sql.connect("app.db") as con:
+		cur = con.cursor()
+		cur.execute("INSERT INTO customer (first_name, last_name, company, email, phone) VALUES (?,?,?,?,?)", (first_name, last_name, company, email, phone))
+		con.commit()
+		new_cur = con.cursor()
+		new_cur.execute("INSERT INTO address (street_address, city, state, country, zip_code) VALUES (?,?,?,?,?)", (street_address, city, state, country, zip_code))

```

Comment:
The customer id for the particular address needs to be inserted into the row as well along with other attributes. As of now your database shows 'NULL' values for the foreign key column and hence the relationship between both the table is broken